recipe/meta.yaml

```
@@ -95,37 +90,34 @@ outputs:
     script: build_output.bat  # [win]
     build:
       number: {{ number }}
-      # cuda.bindings needs to be built with the matching major.minor headers
-      {% if not (environ.get("cuda_compiler_version")|string()) == (bindings_major_version ~ "." ~ bindings_minor_version) %}
-      skip: true
-      {% else %}
       skip: true  # [not build_bindings]
-      {% endif %}
       script_env:
         # Ensure that CUDA includes can be found by the host compiler
-        - CUDA_HOME={{ PREFIX ~ subdir ~ os.pathsep ~ BUILD_PREFIX ~ subdir }}
+        - CUDA_HOME={{ PREFIX ~ subdir }}
       run_exports:
         - {{ pin_subpackage('cuda-python', min_pin='x', max_pin='x') }}
       ignore_run_exports_from:
-        - {{ compiler('cuda') }}
         # The run-exports from these dev packages are too tight, so we ignore them.
         # The cuda-python package supports CUDA Enhanced Compatibility through its
         # use of dlopen, so it is compatible with other CUDA minor versions.
+        - cuda-nvcc-impl
         - cuda-cudart-dev
+        - cuda-cudart-static
         - cuda-nvrtc-dev
     requirements:
       build:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
-        - {{ compiler('cuda') }}
         - {{ stdlib('c') }}
         - cross-python_{{ target_platform }}  # [build_platform != target_platform]
         - python                              # [build_platform != target_platform]
         - cython                              # [build_platform != target_platform]
-        - cuda-version {{ bindings_major_version }}.{{ bindings_minor_version }}
       host:
+        # cuda.bindings needs to be built with the matching major.minor headers
         - cuda-version {{ bindings_major_version }}.{{ bindings_minor_version }}
+        - cuda-nvcc-impl
```

What are we using from `cuda-nvcc-impl` that needs to be in `requirements/host`?

---

This depends on `cuda-nvcc-dev_<arch>` which depends on `cuda-crt-dev_<arch>` which provides the crt headers that we need. `cuda-nvcc-impl` also brings in cudart headers via its other dependencies FWIW. As [noted above](https://github.com/conda-forge/cuda-python-feedstock/pull/118/files#r2076618862), we just need the headers to build, so this seems to be the minimal requirement.

---

(I'd be happy to just depend on the `*_<arch>` packages if there's an easy way to express `<arch>`. My conda knowledge might have become rusty, but I can't think of any simple way atm 😛

---

Would recommend leaving threads unresolved when following up and asking questions so those asked have an opportunity to see them and answer questions

Simply adding [the `cuda-crt-dev_{{ target_platform }}` package]( https://github.com/conda-forge/cuda-nvcc-impl-feedstock/blob/b652c8e4005bbae97c9781affa657f570cb905dd/recipe/meta.yaml#L228 ) like so would work

```suggestion
        - cuda-crt-dev_{{ target_platform }}
```